cfg/17.06/definitions.yaml

```diff
@@ -767,6 +766,7 @@ groups:
     scored: true
 
   - id: 3.17
+    type: manual
     description: "Ensure that daemon.json file ownership is set to root:root (Scored)"
```

Comment:
I'm not sure why this one is becoming a manual test? Iirc that is supposed to be reserved for tests where it can't really be automated.

Reply:
@lizrice 
The daemon.json file is the docker daemon configuration file, which not necessarily exists.
afaiu the check 3.17 verifies the root:root ownership property **assuming existence of the daemon.json file**.

I was not sure what should be the expected output in case the file does not exist. Whether it needs to be FAIL or PASS, so for now the check output is a WARN giving the user to decide whether it passed check or not.

I can remove the "manual" checking, but need to decide what should be the output in case of daemon.json does not exist. Should it be FAIL?

Reply:
If the file doesn't exist then the test should pass (there is no security risk about non-privileged access to / ownership of a file if it doesn't exist). 

We have some similar tests in kube-bench e.g. 1.4.x tests in [1.8/master.yaml](https://github.com/aquasecurity/kube-bench/blob/master/cfg/1.8/master.yaml).  I'm not sure that we need the "not-exist" part in the config file to make this happen. @ttousai could you clarify please?